spec/models/assignment_team_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe AssignmentTeam, type: :model do
+  # pending "add some examples to (or delete) #{__FILE__}"
+
+  let(:team){AssignmentTeam.new name: "Wikipedia contribution_Team2", parent_id: 754, type: "AssignmentTeam", submitted_hyperlinks: "---\n- http://water.com\n- http://shed.com" }
```

Comment:
Hi guys, please read bullet points 5 under "On all refactoring" in your OSS project write-up. In the testing db, there is only very limited data. You cannot assume assignment 754 is in the testing db.